### PR TITLE
feat(core): Introduce worker metrics

### PR DESCRIPTION
**Pull request by ivov**
https://linear.app/n8n/issue/PAY-1734

### CODE REVIEW

File: packages/cli/src/metrics/__tests__/prometheus-metrics.service.test.ts [hunk 1]
```diff
@@ -204,5 +208,18 @@ describe('PrometheusMetricsService', () => {
 			expect(promClient.Counter).toHaveBeenCalledTimes(0); // cache metrics
 			expect(eventService.on).not.toHaveBeenCalled();
 		});
+
+		it('should not set up queue metrics if enabled but instance is not main', async () => {
+			config.set('executions.mode', 'regular');
```
Comment (despairblue) [inline packages/cli/src/metrics/__tests__/prometheus-metrics.service.test.ts:213]:
```suggestion
			config.set('executions.mode', 'queue');
```

**Review by despairblue — CHANGES_REQUESTED** [review]
I only found one issue in a test, the rest looks good 🚀

File: packages/cli/src/metrics/__tests__/prometheus-metrics.service.test.ts (hunk 1 above)
Comment (ivov) [reply packages/cli/src/metrics/__tests__/prometheus-metrics.service.test.ts]: Thank you, good catch!

**Review by despairblue — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.60.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.60.0)

**Pull request merged**